src/azure-cli/azure/cli/command_modules/cdn/_help.py

```diff
@@ -159,6 +159,293 @@
             --enable-compression
 """
 
+helps['cdn endpoint rule'] = """
+type: group
+short-summary: Manage delivery rules for an endpoint.
+"""
+
+helps['cdn endpoint rule add'] = """
+type: command
+short-summary: Add a delivery rule to a CDN endpoint.
+parameters:
+  - name: --rule-name
+    type: string
+    short-summary: Name of the rule.
+  - name: --order
+    type: string
+    short-summary: The order of the rule. The order number must start from 0 and consecutive. Rule with higher order will be applied later.
+  - name: --match-variable
+    type: string
+    short-summary: Name of the match condition.
+  - name: --operator
+    type: string
+    short-summary: Operator of the match condition.
+  - name: --selector
+    type: string
+    short-summary: Selector of the match condition.
+  - name: --match-values
+    type: string
+    short-summary: Match values of the match condition (comma separated).
+  - name: --negate-condition
+    type: string
+    short-summary: Describes if this is negate condition or not.
+  - name: --transform
+    type: string
+    short-summary: Describes what transforms are applied before matching. Possible values are Lowercase, Uppercase.
+  - name: --action-name
+    type: string
+    short-summary: Name of the action.
+  - name: --cache-behavior
+    type: string
+    short-summary: Caching behavior for the requests. Possible values are BypassCache, Override, SetIfMissing.
+  - name: --cache-duration
+    type: string
+    short-summary: The duration for which the content needs to be cached. Allowed format is [d.]hh:mm:ss.
+  - name: --header-action
+    type: string
+    short-summary: Action to perform on headers. Possible values are Append, Overwrite, Delete.
+  - name: --header-name
+    type: string
+    short-summary: Name of header.
+  - name: --header-value
+    type: string
+    short-summary: Value of header.
+  - name: --header-action
```

Comment:
dupe of https://github.com/Azure/azure-cli/pull/11724/files#diff-493400ccbcc5c376fb935fa125dcfa62R204 in line 204?